emulate/emulate_queue.py

```diff
@@ -139,14 +138,29 @@ def run_code(code: str, duration_of_emulation_in_seconds: int) -> None:
 		logging.info(f'Process finished with return code {return_process_poll}')
 
 
-def retrieve_next_animation() -> Code:
+def retrieve_animation_from_priority_queue() -> Code:
+	priority_queue = PriorityQueue.objects.order_by('priority').last()
```

Comment:
taka ciekawostka, można zrobić order_by('-priority').first()
Nie musisz zmieniać, tylko sugeruje - jak wyobrazam sobie kolejke to zawsze wyciągamy "pierwszy element z kolejki"